Modelica/Electrical/Polyphase/Examples/TransformerYY.mo

```diff
@@ -40,7 +40,8 @@ model TransformerYY "Test example with polyphase components"
         transformation(extent={{-30,-100},{-10,-80}})));
   Basic.Resistor transformerR(m=m, R=fill(RT, m)) annotation (Placement(
         transformation(extent={{0,10},{20,30}})));
-  Basic.Inductor transformerL(m=m, L=fill(LT, m)) annotation (Placement(
+  Basic.Inductor transformerL(m=m, L=fill(LT, m),
+    iL(start=zeros(m), fixed=cat(1,fill(true,m-1),{false}))) annotation (Placement(
```

Comment:
But nto here, since it cannot be done for fixed I don't see a good reason to do it for start.